src/sonic-yang-models/yang-models/sonic-system-ldap.yang

```diff
@@ -0,0 +1,128 @@
+module sonic-system-ldap {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-system-ldap";
+    prefix ssys;
+
+    import ietf-inet-types {
+        prefix inet;

```

Comment:
Are we not supporting ldap map configurations ? I do not see the corresponding yang fields 

map passwd loginShell "/bin/bash"
map group member uniqueMember

Reply:
In this first LDAP PRs, we are covering the basic functionalities.
LDAP map and filter are not part of the HLD presented,
it can be part of future work, 
if you need you can raise a request and we can see who can lead this enhancement.